clients/admin-ui/src/features/user-management/UserManagementTabs.tsx

```diff
@@ -36,10 +36,19 @@ const UserManagementTabs = ({
       label: "Permissions",
       content: (
         <Flex gap="97px">
-          <Box w={{ base: "100%", md: "65%", xl: "55%" }}>
+          <Box w={{ base: "100%", md: "50%", xl: "50%" }}>
             <PermissionsForm />
           </Box>
-          <Box w={{ base: "100%", md: "35%", xl: "45%" }}>
+          <Box
+            position="absolute"
+            top="114px"
+            right="24px"
+            overflowY="scroll"
+            padding="24px"
+            w="35%"
+            borderLeftWidth="1px"
+            borderBottomWidth="1px"
```

Comment:
the designs don't look like they have a bottom border? can we try giving this a `height="100%"` so that the vertical line will go all the way down, and then we wouldn't need a bottom border?

Reply:
this was a tricky one, using `height="100%"` only made the height extend to the viewable screen, not below that. I'll see if I can work around this someone.

Reply:
For now I've allowed some extra height to be added to the page such that assigning a couple systems to users does not cause the vertical line to be shorter than the page:

```
height="calc(100% + 100px)"
```

This is not dynamic based on how many systems are assigned to the user, which is one downfall of absolutely positioned elements. 

Ideally this vertical line element would live on the permissions drawer element, but since the line extends above the permissions drawer element, it would have to be absolutely positioned as well, and it would pose greater challenges for calculating exact position.

I'm open to better ideas @allisonking.